packages/did-core-test-server/suites/utils.js

```diff
@@ -105,6 +105,52 @@ const isValidVerificationMethod = (vm) => {
   return true;
 }
 
+function updateContextsToDidv11(obj) {
+  if(typeof obj !== 'object' || obj === null) {
+    return;
+  }
+
+  for(const key in obj) {
+    if(key === '@context') {
+      if(obj['@context'][0] ===
+        'https://www.w3.org/ns/did/v1') {
+          obj['@context'][0] = 'https://www.w3.org/ns/did/v1.1';
+      }
+    }
+
+    if(key === 'representation') {
+      let newRepresentation = JSON.parse(obj['representation']);
+      if(newRepresentation['@context'] !== undefined) {
+        if(newRepresentation['@context'][0] ===
+          'https://www.w3.org/ns/did/v1') {
+            newRepresentation['@context'][0] = 'https://www.w3.org/ns/did/v1.1';
+        }
+        obj['representation'] = JSON.stringify(newRepresentation);
+      }
+    } else {
+      if(typeof obj[key] === 'object' && obj[key] !== null) {
+        updateContextsToDidv11(obj[key]);
+      }
+    }
+  }
+}
+
+const addDidv11Implementations = (implementations) => {
+  const allImplementations = [];
+
+  implementations.forEach((suiteConfig) => {
+    v10SuiteConfig = suiteConfig;
+    v11SuiteConfig = JSON.parse(JSON.stringify(suiteConfig));
```

Comment:
Is it ok to use the standard clone api here?
```suggestion
    v11SuiteConfig = structuredClone(suiteConfig);
```

Reply:
Yes. I was unsure if it was available in node.js 20, which we have to use for this test suite. Looks like it's been in node.js since node.js 17.